develop/dev-guide-bi-looker-stduio.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Connect to TiDB Cloud with Looker Studio
+summary: Learn how to connect to TiDB Cloud using Looker Studio
+---
+
+# Connect to TiDB Cloud with Looker Studio
+
+TiDB is a MySQL-compatible database, and [Looker Studio](https://lookerstudio.google.com/) is a free web-based BI tool that can visualize data from various sources. 
+
+In this tutorial, you can learn how to connect to your TiDB cluster using Looker Studio.
+
+> **Note:**
+>
+> - This tutorial is compatible with TiDB Serverless.
+
+## Prerequisites
+
+To complete this tutorial, you need:
+
+- Google Account that can use Looker Studio.
+- A TiDB Serverless cluster.
```

Comment:
Doesn't this also work with a dedicated cluster?

Reply:
Maybe say something like "For this tutorial we will use a serverless cluster. Looker Studio can also be used with dedicated clusters."

Reply:
@dveeden Thanks, that should work with a dedicated cluster as well. I will check and add TiDB Dedicated Tab.